src/pages/test-report/cat-c/test-report.cat-c.page.ts

```diff
@@ -283,4 +292,6 @@ export class TestReportCatCPage extends BasePageComponent {
     this.store$.dispatch(new AddDangerousFault(Competencies.responseToSignsTrafficLights));
     this.navController.push(CAT_C.DEBRIEF_PAGE);
   }
+
+  showUncoupleRecouple = (): boolean => this.testCategory !== TestCategory.C;
```

Comment:
I would check which category is, rather than which one isn't

so: `=> this.testCategory === TestCategory.CE || this.testCategory === TestCategory.C1E;`